src/main/resources/application-dev.yaml

```diff
@@ -15,12 +15,12 @@ spring:
       enable-auto-commit: true
       auto-commit-interval: 1000
   jpa:
-    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
     hibernate:
       ddl-auto: none
     properties:
       hibernate:
-        dialect: org.hibernate.dialect.PostgreSQL10Dialect
+        dialect: org.hibernate.dialect.PostgreSQLDialect
```

Comment:
Please check if `database-platform` and `dialect` could be removed. I read somewhere that there is autodetection exists in the new version of Spring.

Reply:
looks like yes, removed them

Reply:
Had to return `dialect` back, because when the db is not available application is not able to determine the dialect and startup fails. As the result health check does not work on jenkins build.